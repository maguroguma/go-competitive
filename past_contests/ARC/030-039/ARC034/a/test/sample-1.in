2
37 54 68 66 802
58 108 106 103 871
