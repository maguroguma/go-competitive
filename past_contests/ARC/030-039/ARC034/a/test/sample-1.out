481.4555555555555555
